src/workloads/query/TimeseriesEnum.yml

```diff
@@ -11,9 +11,9 @@ Keywords:
 GlobalDefaults:
   Database: &database test
   TSCollection: &tsCollection Collection0
-  DocumentCount: &documentCount 1e7
+  DocumentCount: &documentCount 1e6
```

Comment:
The timeout was caused by each individual query running for too long, so I reduced the size of the data set. The queries still take many seconds to execute, which is more than enough to get good signal. I also reduced the repeat count, since the task currently takes >2 hours, which seems unnecessarily long.